tests/test_butler.py

```diff
@@ -55,6 +57,29 @@ def _get_datasets_from_chain(self, chain, datasetType=...):
                                                                                      collections=run)}
         return unresolved
 
+    def testMetadata(self):
+        """Test that metadata can be retrieved."""
+
+        dataId = {"instrument": "HSC", "visit": 903342, "detector": 10}
+        collection = "demo_collection"
+
+        cal = self.butler.get("calibrate_metadata", dataId=dataId, collections=collection)
+        isr = self.butler.get("isr_metadata", dataId=dataId, collections=collection, exposure=903342)
+        charImage = self.butler.get("characterizeImage_metadata", dataId=dataId, collections=collection)
+
+        # Confirm that the calibrate metadata is a different type to the
+        # other two metadata objects (calibrate_metadata should be PropertySet
+        # and the others TaskMetadata).
```

Comment:
Remove or alter this comment: it's redundant with the tests below. Maybe "Confirm that only the calibrate metadata is a PropertySet" or something like that?